src/Tasks.UnitTests/CreateProperty_Tests.cs

```diff
@@ -47,7 +47,7 @@ public void CreateBlankProperty()
 
         /// <summary>
         /// Make sure that I can use the CreateProperty task to create a property
-        /// that has a parseable semicolon in it.
+        /// that has a parsable semicolon in it.
```

Comment:
Was skeptical of this change but it does look like a sorta-standard: https://en.wiktionary.org/wiki/parsable.